projects/kit/components/pdf-viewer/pdf-viewer-options.ts

```diff
@@ -2,6 +2,7 @@ import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
 
 export interface TuiPdfViewerOptions<I = undefined> {
     readonly label: string;
+    readonly errorMessage?: string;
```

Comment:
Let's make it as just `error` (with type `PolymorpheusContent`).

Our users can decide to put some custom content.
For example, error-page component.